proto/redpanda/api/dataplane/v1alpha2/topic.proto

```diff
@@ -292,10 +292,7 @@ service TopicService {
     };
   }
   rpc GetTopicConfigurations(GetTopicConfigurationsRequest) returns (GetTopicConfigurationsResponse) {
-    option (google.api.http) = {
-      get: "/v1alpha2/topics/{topic_name}/configurations"
-      response_body: "configurations"
-    };
+    option (google.api.http) = {get: "/v1alpha2/topics/{topic_name}/configurations"};
```

Comment:
Should we do it, despite knowing the number of entries is very limited and will never exceed an unreasonable number that couldn't be retrieved in one go?